src/opcodes/tests/test_disabled.cairo

```diff
@@ -0,0 +1,90 @@
+use core::array::ArrayTrait;
+use shinigami::opcodes::tests::utils;
+use shinigami::errors::Error;
+
+// TODO is there a way to define this as a const?
```

Comment:
Creating a constant array isn't possible currently. This is probably the best way to handle this for now.

Maybe in the future, when the interpreter/script checker is built out, we can define `disabled_opcodes` as a member variable, set it once, and use it in the tests.

Reply:
Makes sense! Thanks

Reply:
For sure, I'll leave the comment for now as a reminder.